appserver/deployment/dol/src/main/java/com/sun/enterprise/deployment/util/ResourceValidator.java

```diff
@@ -127,7 +125,7 @@ public void postConstruct() {
     @Override
     public void event(Event event) {
         if (event.is(Deployment.AFTER_APPLICATION_CLASSLOADER_CREATION)) {
-            dc = (DeploymentContext) event.hook();
+            DeploymentContext dc = (DeploymentContext) event.hook();
```

Comment:
Avoid names like "dc", use "deploymentContext", or "context" or something like that here